core/src/spec/spec.rs

```diff
@@ -350,7 +373,7 @@ impl Spec {
 }
 
 /// Load from JSON object.
-fn load_from(s: cjson::spec::Spec) -> Result<Spec, Error> {
+fn load_from(s: cjson::spec::Spec, handlers: &[Arc<ActionHandler>]) -> Result<Spec, Error> {
```

Comment:
IMO `handlers` must be Vec. Passing it as slice and converting it to `Vec` is redundant.